FBSnapshotTestCase/Categories/UIImage+Snapshot.m

```diff
@@ -57,6 +62,11 @@ + (UIImage *)fb_imageForView:(UIView *)view
 
   UIImage *snapshot = UIGraphicsGetImageFromCurrentImageContext();
   UIGraphicsEndImageContext();
+
+  if (removeFromSuperview) {
```

Comment:
I disagree, since there's an edge case where the condition you're testing is actually setup by the user, and therefore would be incorrect for us to tear down. The intent here is to only tear down what we ourselves setup on line 50/55.